katka/views.py

```diff
@@ -41,3 +44,9 @@ class CredentialSecretsViewSet(AuditViewSet):
     def get_queryset(self):
         user_groups = self.request.user.groups.all()
         return super().get_queryset().filter(credential__team__group__in=user_groups)
+
+
+class SCMServiceViewSet(ReadOnlyAuditViewMixin):
+    model = SCMService
+    serializer_class = SCMServiceSerializer
+    permission_classes = [IsAuthenticated]
```

Comment:
Good point. I don't really think it is, but at the same point I feel quite uncomfortable leaving an endpoint public (even though it will be protected by katka-api). I tend to think (after talking to Jeroen) that we should add this to the base viewset even. What are your thoughts on that?

Reply:
I think it makes sense for now, if we ever want to allow anonymous access, we can always change it later.

Reply:
ok, I'll merge this one then